multisrc/overrides/zeistmanga/datgarscanlation/src/DatGarScanlation.kt

```diff
@@ -0,0 +1,32 @@
+package eu.kanade.tachiyomi.extension.es.datgarscanlation
+
+import eu.kanade.tachiyomi.multisrc.zeistmanga.ZeistManga
+import org.jsoup.nodes.Document
+
+class DatGarScanlation : ZeistManga("DatGarScanlation", "https://datgarscanlation.blogspot.com", "es") {
+
+    private val altChapterFeedRegex = """label\s=\s'([^']+)'""".toRegex()
```

Comment:
Make whitespace optional
```suggestion
    private val altChapterFeedRegex = """label\s*=\s*'([^']+)'""".toRegex()
```